app/views/facility_journals/_problem_transactions.html.haml

```diff
@@ -0,0 +1,68 @@
+.row
+  .span12
+    %p.alert.alert-error= t("facility_journals.new.expired_payment")
+.row
```

Comment:
I borrowed a ton of this from `shared/transactions/_table_inside`--not sure what else, if anything, we might want to strip out.